doc/source/whatsnew/v0.24.0.txt

```diff
@@ -168,6 +168,120 @@ Current Behavior:
     ...
     OverflowError: Trying to coerce negative values to unsigned integers
 
+read_html Incompatibilities
+---------------------------
+
+:func:`read_html` previously ignored ``colspan`` and ``rowspan`` attributes.
+Now it understands them, treating them as a sequence of cells with the same
+value.
+
+Previous Behavior:
+
+.. code-block:: ipython
+
+    In [1]: pd.read_html("""
+          <table>
+            <thead>
+              <tr>
+                <th>A</th><th>B</th><th>C</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td colspan="2">1</td><td>2</td>
+              </tr>
+            </tbody>
+          </table>
+        """)
+    Out [1]:
+    [   A  B   C
+    0  1  2 NaN]
+
+Current Behavior:
+
+.. code-block:: ipython
+
+    In [1]: pd.read_html("""
+          <table>
+            <thead>
+              <tr>
+                <th>A</th><th>B</th><th>C</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td colspan="2">1</td><td>2</td>
+              </tr>
+            </tbody>
+          </table>
+        """)
+    Out [1]:
+    [   A  B  C
+    0  1  2  2]
+
+Calls that relied on the previous behavior will need to be changed.
+
+Also, :func:`read_html` previously ignored some ``<tr>`` elements when called
```

Comment:
See what @jreback thinks but I don't consider this second half necessary as it addresses a minor bug. Otherwise lgtm

Reply:
yeah you can just list as a 1-liner in the io section is ok, no example for this needed (the latter half)